build-support/scripts/up

```diff
@@ -30,7 +30,9 @@ echo "Connected!"
 
 # Apply limited license
 echo -n "Applying license.. "
-docker exec -e "NUODB_LIMITED_LICENSE_CONTENT=$NUODB_LIMITED_LICENSE_CONTENT" ad1 /bin/bash -c 'echo "$NUODB_LIMITED_LICENSE_CONTENT" | base64 -d > ~/nuodb.lic && nuocmd set license --license-file ~/nuodb.lic && rm ~/nuodb.lic'
+docker exec -e "NUODB_LIMITED_LICENSE_CONTENT=${NUODB_LIMITED_LICENSE_CONTENT}" ad1 /bin/bash -c 'echo "${NUODB_LIMITED_LICENSE_CONTENT}" | base64 -d > ~/nuodb.lic && nuocmd set license --license-file ~/nuodb.lic && rm ~/nuodb.lic'
```

Comment:
It looks like the only change here is to replace `$NUODB_LIMITED_LICENSE_CONTENT` with `${NUODB_LIMITED_LICENSE_CONTENT}`. Why would that matter?

Reply:
it should not matter, but it was an attempt to see if the reason that line fails for me would change anything.  I am not able to run that line successfully, but Eusebio can, and the overnights can, which is why the other commented code is being left in.